demos/checkbox.html

```diff
@@ -201,6 +202,10 @@
         document.getElementById('make-ind').addEventListener('click', function() {
           checkboxInstance.indeterminate = true;
         });
+
+        document.getElementById('toggle-disabled').addEventListener('click', function() {
```

Comment:
That is exactly what I was expecting! 😃 I think that every class should have its setters that should be in charge of these changes to the root element. 🎉 🎉